r/R/type.R

```diff
@@ -460,7 +472,10 @@ NANOARROW_TYPE <- list(
   LARGE_STRING = 35L,
   LARGE_BINARY = 36L,
   LARGE_LIST = 37L,
-  INTERVAL_MONTH_DAY_NANO = 38L
+  INTERVAL_MONTH_DAY_NANO = 38L,
+  RUN_END_ENCODED = 39L,
```

Comment:
I didn't expect to see the additioin of `RUN_END_ENCODED` here. Just noted while I do my review.

Reply:
👌